### PR TITLE
Add 2.22.0 release

### PR DESCRIPTION
**Pull request by denisok**
Add community preview for 2.22.0

### CODE REVIEW

**Comment by denisok** [comment]
@rnovikovP please check

File: content/blog/2021-09-16-2-22-0-preview.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+### AMI 
+
+https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/server/aws.html
+
+Artifact id: `ami-0a6b861c9225afbd8`
+
+---
+
+Please also check out our Engineering Monthly Meetings https://percona.community/contribute/engineeringmeetings/ and join us on our journey in OpenSource! Contact us in [per.co.na/discord](https://per.co.na/discord).
```
Comment (shoffman-percona) [inline content/blog/2021-09-16-2-22-0-preview.md:60]: Need to hint to people what to do when done testing?  
disable testing repo...roll back client version
maybe recommend test in a sandbox/vm vs on your live system?

File: content/projects/pmm.md [hunk 2]
```diff
@@ -78,5 +78,20 @@ For more details, please refer to our [Architecture](https://www.percona.com/doc
 
 If you're interested in contributing, please check out our page on [contributing](/contribute) as well as the [main PMM repository](https://github.com/percona/pmm) that also lists all our exporter repositories.
 
+## PMM Roadmap and Releases
+
+Please check what is in PMM Roadmap on this [Dashboard](https://jira.percona.com/secure/Dashboard.jspa?selectPageId=13502), choose "PMM" in Roadmap picker.
```
Comment (shoffman-percona) [inline content/projects/pmm.md:83]:
```suggestion
Please check out the PMM Roadmap on this [Dashboard](https://jira.percona.com/secure/Dashboard.jspa?selectPageId=13502), choose the project "Percona Monitoring and Management (PMM)" in All Product Issues/Roadmap picker.
```

File: content/blog/2021-09-16-2-22-0-preview.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+
+## 2.20.0 Preview Release
+
+Percona Monitoring and Management 2.22.0 is released today as a Community preview.
+
+PMM team really appreciates your feedback!
+
+Known issue:
+https://jira.percona.com/browse/PMM-8829
```
Comment (shoffman-percona) [inline content/blog/2021-09-16-2-22-0-preview.md:20]:
```suggestion
- [PMM-8829](https://jira.percona.com/browse/PMM-8829) - "Missing Listen Port" error for external exporters after restart
```

File: content/blog/2021-09-16-2-22-0-preview.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+Release notes:
+https://deploy-preview-588--pmm-doc.netlify.app/release-notes/2.22.0.html
```
Comment (shoffman-percona) [inline content/blog/2021-09-16-2-22-0-preview.md:23]:
```suggestion
Release Notes Preview found [here](https://deploy-preview-588--pmm-doc.netlify.app/release-notes/2.22.0.html)
```

File: content/blog/2021-09-16-2-22-0-preview.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+If you want to install pmm2-client package, please enable testing repository via Percona-release: 
+```
+percona-release enable original testing
+```
+
+install pmm2-client package for your OS via package manager
+
+### OVA 
+
+https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/server/virtual-appliance.html
```
Comment (shoffman-percona) [inline content/blog/2021-09-16-2-22-0-preview.md:48]:
```suggestion
[Instructions](https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/server/virtual-appliance.html)
```

File: content/blog/2021-09-16-2-22-0-preview.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+Known issue:
+https://jira.percona.com/browse/PMM-8829
+
+Release notes:
+https://deploy-preview-588--pmm-doc.netlify.app/release-notes/2.22.0.html
+
+
+### PMM server docker
+
+https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/server/docker.html
```
Comment (shoffman-percona) [inline content/blog/2021-09-16-2-22-0-preview.md:28]:
```suggestion
[Instructions](https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/server/docker.html)
```

**Review by shoffman-percona — CHANGES_REQUESTED** [review]
some cleanup suggestions

File: content/blog/2021-09-16-2-22-0-preview.md (hunk 1 above)
Comment (denisok) [reply content/blog/2021-09-16-2-22-0-preview.md:62]: added a hint

**Review by shoffman-percona — APPROVED** [review]

File: content/blog/2021-09-16-2-22-0-preview.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+  - denys_kondratenko
+images:
+  - superhero.png
+slug: preview-release
+---
+
+
+## 2.20.0 Preview Release
+
+Percona Monitoring and Management 2.22.0 is released today as a Community preview.
```
Comment (PaulJacobs-percona) [inline content/blog/2021-09-16-2-22-0-preview.md:15]: What does 'Community preview' mean? How is it different to a normal release?

File: content/blog/2021-09-16-2-22-0-preview.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+PMM team really appreciates your feedback!
+
+Please try it out only in **testing environment**, those packages and images aren't supposed to be deployed in production systems. Final version is expected to be released through the standard channels in a coming week.
```
Comment (PaulJacobs-percona) [inline content/blog/2021-09-16-2-22-0-preview.md:19]:
```suggestion
We encourage you to try this PMM community release in **testing environments** only, as these packages and images are not fully production-ready. The final version is expected to be released through the standard channels in the coming week.
```

**Review by PaulJacobs-percona — DISMISSED** [review]

File: content/blog/2021-09-16-2-22-0-preview.md (hunk 7 above)
Comment (denisok) [reply content/blog/2021-09-16-2-22-0-preview.md:15]: no idea, this is first try. we will come up with something better.
it is like Early Preview probably.

Comment (denisok) [reply content/blog/2021-09-16-2-22-0-preview.md:15]: what is the better name ?

Comment (rnovikovP) [reply content/blog/2021-09-16-2-22-0-preview.md:15]:
```suggestion
Percona Monitoring and Management 2.22.0 is released today as a Preview Release.
```

File: content/blog/2021-09-16-2-22-0-preview.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+---
+
+
+## 2.20.0 Preview Release
+
+Percona Monitoring and Management 2.22.0 is released today as a Community preview.
+
+PMM team really appreciates your feedback!
+
+We encourage you to try this PMM community release in **testing environments** only, as these packages and images are not fully production-ready. The final version is expected to be released through the standard channels in the coming week.
```
Comment (rnovikovP) [inline content/blog/2021-09-16-2-22-0-preview.md:19]:
```suggestion
We encourage you to try this PMM Preview Release in **testing environments** only, as these packages and images are not fully production-ready. The final version is expected to be released through the standard channels in the coming week.
```

**Comment by denisok** [comment]
@dbazhenov could you please advertise this after it lands ?

**Review by rnovikovP — APPROVED** [review]
LGTM

**Pull request merged**